### PR TITLE
docs: Mention differences with `conda`

### PR DESCRIPTION
**Pull request by corneliusroemer**
It would be very useful for users to know which `conda` features `(micro)mamba` does not support. There's a section on "additional features" already in the readme.

This documents issue https://github.com/mamba-org/mamba/issues/803

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ It can significantly reduce your CI setup time by:
 - Caching package downloads.
 - Caching entire `conda` environments.
 
+## Limitations compared to `conda`
```
Comment (jjerphan) [inline README.md:63]: As there might be features or behavior of conda which we might not port.
```suggestion
## Differences with `conda`
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,12 @@ It can significantly reduce your CI setup time by:
...
+
+While `mamba` and `micromamba` are generally a drop-in replacement for `conda` there are a small number of limitations:
```
Comment (jjerphan) [inline README.md:65]:
```suggestion
While `mamba` and `micromamba` are generally a drop-in replacement for `conda` there are some differences:
```

File: README.md [hunk 3]
```diff
@@ -60,6 +60,12 @@ It can significantly reduce your CI setup time by:
...
+
+- No support of revisions
```
Comment (jjerphan) [inline README.md:67]:
```suggestion
- mamba and micromamba do no support revisions yet (for discussions, see https://github.com/mamba-org/mamba/issues/803)
```

**Review by jjerphan — COMMENTED** [review]
Thank you for this PR, @corneliusroemer.

I propose to have this PR also address https://github.com/mamba-org/mamba/issues/3362. What do you think?

**Review by JohanMabille — APPROVED** [review]

**Pull request merged**